components/calendar/src/cal/hijri.rs

```diff
@@ -214,10 +214,14 @@ impl Rules for AstronomicalSimulation {
     }
 }
 
-/// [`Hijri`] [`Rules`] for the [Umm al-Qura](https://en.wikipedia.org/wiki/Islamic_calendar#Saudi_Arabia's_Umm_al-Qura_calendar) calendar.
+/// [`Hijri`] [`Rules`] for the [Umm al-Qura](https://en.wikipedia.org/wiki/Islamic_calendar#Saudi_Arabia's_Umm_al-Qura_calendar) calendar from the start of 1300 AH to the end of 1600 AH.
```

Comment:
nit: on the Chinese docs we put the exact periods in the next paragraph. the first paragraph of a rustdoc should be a short sentence describing the type

Reply:
Removed the detail.